index.html

```diff
@@ -34,18 +35,52 @@ <h1>Flight List</h1>
 
         </table>
       </div>
+=======
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+>>>>>>> Stashed changes
+
+
+    <link rel="stylesheet" href="./src/style.css" />
+    <link rel="stylesheet" href="/dist/output.css">
```

Comment:
Depending if you are using postcss or watch you should only need one stylesheet.